src/CryptoWorker.ts

```diff
@@ -0,0 +1,28 @@
+import { expose } from "threads"
+import * as crypto from 'crypto'
+
+let cipher: crypto.Cipher
+let decipher: crypto.Decipher
+
+const cryptoWorker = {
+  init(algorithm: string, key: Buffer, initVector: Buffer): void {
+		cipher = crypto.createCipheriv(algorithm, key, initVector)
```

Comment:
There is literal tab characters. They should be spaces.

Reply:
Check if your editor is respecting the `.editorconfig`. If you don't have one, get one from `JavaScript-Demo` repo.

Reply:
All files reformatted according to .editorconfig